src/app/clusters/level-control/level-control.cpp

```diff
@@ -1287,6 +1292,11 @@ void emberAfLevelControlClusterServerInitCallback(EndpointId endpoint)
     emberAfPluginLevelControlClusterServerPostInitCallback(endpoint);
 }
 
+void MatterLevelControlClusterServerShutdownCallback(EndpointId endpoint)
+{
+    DeviceLayer::SystemLayer().CancelTimer(timerCallback, reinterpret_cast<void *>(static_cast<uintptr_t>(endpoint)));
```

Comment:
Why not call `cancelEndpointTimerCallback` here?

Reply:
agree